pkgs/http/README.md

```diff
@@ -100,3 +100,15 @@ and increases the delay by 1.5x each time. All of this can be customized using
 the [`RetryClient()`][new RetryClient] constructor.
 
 [new RetryClient]: https://pub.dev/documentation/http/latest/retry/RetryClient/RetryClient.html
+
+## Response Headers
+
+The Dart HTTP client by default allows only the CORS-safelisted response headers:
```

Comment:
This is for BrowserClient-only, right? I'm not sure that this is the best place to add this documentation.

The documentation would logically belong to BrowserClient but people are unlikely to look there.

Maybe add a note in the [`headers` field](https://pub.dev/documentation/http/latest/http/BaseResponse/headers.html).